lib/samlp.js

```diff
@@ -196,6 +201,8 @@ function getLogoutResponse (options, callback) {
   });
 
   if (options.signResponse) {
+    options.prefix = options.prefix && (typeof options.prefix === 'string') ? options.prefix : '' ;
```

Comment:
`(typeof options.prefix === 'string')` instead of `options.prefix && (typeof options.prefix === 'string')` ?